src/js/ctrl/export.js

```diff
@@ -55,7 +56,12 @@ exporter.dataset = function(state, internal, id) {
       json2csv({data: values, del: format === 'tsv' ? '\t' : ','}) : values;
   }
 
-  return spec;
+  if(sort == undefined) {
+    return spec;
+  } else {
+    spec['transform'] = sort;
```

Comment:
`spec.transform` will suffice here. Stylistically, we prefer to only use square bracket notation when the key is an invalid identifier (e.g., contains symbols).